src/components/Tooltip.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  .tooltip {
```

Comment:
Rather than defining a bunch of classes here (which I _believe_ will pollute the global CSS class namespace... there's a chance I'm wrong on that though?), can't we just attach these styles to a few styled components and make this cleaner?

e.g.:

```javascript
const Tooltip = styled.div`
  position: relative;
  display: inline-block;
  overflow: visible !important
`;
```

styled components supports CSS selectors, so we should be fine doing

```javascript
const TooltipText = styled.span`
  visibility: hidden;
  ...
  
  ::after {
    content: "";
    ...
  }
`;
```